blockprod/src/rpc.rs

```diff
@@ -68,6 +68,16 @@ impl BlockProductionRpcServer for super::BlockProductionHandle {
         )
     }
 
+    /// Generate a block with the given transactions.
+    /// Parameters:
```

Comment:
Would add a blank line between the two so it does not look weird in generated docs